Sources/Display/Platform/Win32/win32_window.cpp

```diff
@@ -1685,8 +1735,6 @@ void Win32Window::create_hid_devices()
 
 void Win32Window::get_styles_from_description(const DisplayWindowDescription &desc, DWORD &style, DWORD &ex_style)
 {
-	style = 0;
-	ex_style = 0;
```

Comment:
Why is this needed? Without those two lines the flags might not be set correctly if a caller hasn't zero-initialized the out arguments.